RFC/src/RFC-0130_Mining.md

```diff
@@ -1 +1,147 @@
+"OPTIONAL" in this document are to be interpreted as described in [RFC 2119](http://tools.ietf.org/html/rfc2119).
+
+## Disclaimer
+
+The purpose of this document and its content is for information purposes only and may be subject to change or update
+without notice.
+
+This document may include preliminary concepts that may or may not be in theprocess of being developed by the Tari
+community. The release of this document is intended solely for review and discussion by the community regarding the
+technological merits of the potential system outlined herein.
+
+## Goals
+
+This document will provide a brief overview of the Tari merged mining process and will introduce the primary 
+functionality required of the Mining Server and Mining Worker.
+
+## Related RFCs
+
+* [RFC-0100: Base layer](RFC-0100_BaseLayer.md)
+* [RFC-0110: Base nodes](RFC-0110_BaseNodes.md)
+
+## Description
+
+### Assumptions
+- That the Tari [blockchain] will be merged mined with Monero.
+- The Tari [Base Layer] has a network of [Base Node]s that verify and propagate valid [transaction]s and [block]s. 
+
+### Abstract
+
+The process of merged mining Tari with Monero on the Tari Base Layer is performed by [Mining Server]s and [Mining 
+Worker]s. Mining Servers are responsible for constructing new blocks by bundling transactions from the [mempool] 
+of a connected Base Node. They then distribute Proof-of-Work(PoW) tasks to Mining Workers in an attempt to solve 
+newly created blocks. Solved solutions and shares are sent by the Mining Workers to the Mining Server, who in turns 
+verifies the solution and distributes the newly created blocks to the Base Node and Monero Node for inclusion in 
+their respective blockchains.
+
+### Merged mining on the Tari Base Layer
+
+This document is divided into three parts. First, a brief overview of the merged mining process and the interactions 
+between the Base Node, Mining Server and Mining Worker will be provided, then the primary functionality required 
+of the Mining Server and Mining Worker will be proposed.
+
+####  Overview of the Tari merged mining process using Mining Servers and Mining Workers
+
+Mining on the Tari Base Layer consists of three primary entities: the Base Nodes, Mining Servers and Mining Workers. 
+A description of the Base Node is provided in [RFC-0110/Base Nodes] (https://tari-project.github.io/tari/RFC-0110_BaseNodes.html).
+A Mining Server is connected locally or remotely to a Tari Base Node and a Monero Node, and is responsible for 
+constructing Tari and Monero Blocks from their respective mempools. The Mining Server should retrieve transactions 
+from the mempool of the connected Base Node and assemble a new Tari block by bundling transactions together.
+
+Mining servers may re-verify transactions before including them in a new Tari block, but this 
+verification process of checking that the transaction's rules such as signatures and timelocks are enforced is the 
```

Comment:
this sentence doesnt read weird. Perhaps refactor to something in the likes of: but this enforcement of 
verification and transaction rules such as signatures and timelocks is the responsibility of the connected Base node.

Reply:
Change applied in new PR.